modules/apps/user/user-test/src/testIntegration/java/com/liferay/user/service/test/UserServiceTest.java

```diff
@@ -48,9 +50,9 @@ public void testAddInvalidOrganizationUsers() throws Exception {
 
 		User user1 = UserTestUtil.addUser();
 
-		Company company = CompanyTestUtil.addCompany();
+		Company company = _companyLocalService.getCompany(TestPropsValues.getCompanyId());
 
-		User user2 = UserTestUtil.addCompanyAdminUser(company);
+		User user2 = UserTestUtil.addUser(company);
 
```

Comment:
Leave as it is, in this case we need a new company because we are checking permissions